### PR TITLE
samples: lvgl: cleanup: sample folder factorisation

### PR DESCRIPTION
**Pull request by everedero**
Move the subsys/display/lvgl sample into the modules/lvgl folder, with renaming to helloworld to avoid lvgl/lvgl path.

### CODE REVIEW

**Comment by fabiobaltieri** [comment]
can you update the test names in `sample.yaml` while at it? They should now say something like `sample.modules.lvgl....`

thanks!

**Review by fabiobaltieri — DISMISSED** [review]

File: samples/modules/lvgl/helloworld/README.rst
Comment (kartben) [inline samples/modules/lvgl/helloworld/README.rst:1]: this needs an entry in `doc/_scripts/redirects.py` -- we want people navigating an old doc (ex. https://docs.zephyrproject.org/3.5.0/samples/subsys/display/lvgl/README.html) to not land on a 404 when clicking "The latest development version of this page may be more current than this released 3.5.0 version."

**Review by kartben — CHANGES_REQUESTED** [review]

**Review by fabiobaltieri — DISMISSED** [review]

File: samples/modules/lvgl/helloworld/README.rst
Comment (kartben) [inline samples/modules/lvgl/helloworld/README.rst:1]: Please update the path to the sample in the "Building and Running" section (2 occurences). Sorry that it skipped my mind in yesterday's review.

**Review by kartben — CHANGES_REQUESTED** [review]

File: doc/_scripts/redirects.py [hunk 1]
```diff
@@ -173,6 +173,7 @@
     ('reference/util/index', 'kernel/util/index'),
     ('samples/drivers/kscan_touch', 'samples/subsys/input/input'),
     ('samples/net/cloud/google_iot_mqtt', 'samples/net/cloud'),
+    ('samples/subsys/display/lvgl', 'samples/modules/lvgl/helloworld'),
```
Comment (kartben) [inline doc/_scripts/redirects.py:176]: actually, the syntax is probably more confusing than it needs to be but it should be:

```suggestion
    ('samples/subsys/display/lvgl/README', 'samples/modules/lvgl/helloworld/README'),
```

**Pull request closed without merging**